docs/DESIGN.md

```diff
@@ -44,23 +72,112 @@ We will also provide a general comment section under each post.
 - Backend
 - Devops
 - Cybersecurity
+- Freelance
+
+### Work Status
+
+- Nothing
+- Open to Work
+- Hiring
+
+### User Type
+
+- Inactive
+- Active
+- Admin
+
 ### Users
-|field|type|
-|:---|:---|
-|created|datetime|
-| lastLogin|datetime|
-|email|string|
-|name|string|
-|userTypeID|number|
+
+As this table is auto generated by Supabase, we only use several fields from it.
```

Comment:
I don't really care that we put it here (it can be easily viewed in the code), but do we really want to? Putting it here means we need to manually update this on every schema change.. (this comment is relevant to everything under Schemas & Collections.